data/json/recipes/basecamps/expansion/recipe_modular_canteen/version_2/recipe_modular_canteen_common.json

```diff
@@ -0,0 +1,911 @@
+      "MAP_ROTATE_180_IF_S",
+      "MAP_ROTATE_270_IF_SW",
+      "MAP_MIRROR_HORIZONTAL_IF_W",
+      "MAP_ROTATE_270_IF_W"
+    ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "fbmk_2_fireplace",
+    "description": "Now that we have some cover, we should build a fireplace for cooking and grab a pot.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbmk_2_fireplace",
+    "blueprint_name": "build a fireplace",
+    "blueprint_requires": [ { "id": "fbmk_2_center" } ],
+    "blueprint_provides": [
+      { "id": "kitchen" },
+      { "id": "fbmk_2_recipes_1" },
+      { "id": "fbmk_2_fire" },
+      { "id": "trapping" },
+      { "id": "hunting" }
+    ],
+    "blueprint_excludes": [ { "id": "fbmk_2_fire" } ],
+    "blueprint_resources": [ "fake_fireplace", "fake_fireplace", "pot" ],
+    "components": [ [ [ "pot", 1 ], [ "rock_pot", 1 ], [ "pot_copper", 1 ], [ "clay_pot", 1 ] ], [ [ "pan", 1 ] ] ],
+    "flags": [
+      "MAP_MIRROR_HORIZONTAL_IF_N",
+      "MAP_ROTATE_90_IF_NE",
+      "MAP_MIRROR_HORIZONTAL_IF_E",
+      "MAP_ROTATE_90_IF_E",
+      "MAP_ROTATE_180_IF_SE",
+      "MAP_MIRROR_HORIZONTAL_IF_S",
+      "MAP_ROTATE_180_IF_S",
+      "MAP_ROTATE_270_IF_SW",
+      "MAP_MIRROR_HORIZONTAL_IF_W",
+      "MAP_ROTATE_270_IF_W"
+    ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "fbmk_2_stove",
+    "description": "Now that we have some cover, we should build a stove for cooking and grab a pot.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "skill_used": "fabrication",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbmk_2_stove",
+    "blueprint_name": "build a wood stove",
+    "blueprint_requires": [ { "id": "fbmk_2_center" } ],
+    "blueprint_provides": [
+      { "id": "kitchen" },
+      { "id": "fbmk_2_recipes_1" },
+      { "id": "fbmk_2_fire" },
+      { "id": "trapping" },
+      { "id": "hunting" }
+    ],
+    "blueprint_excludes": [ { "id": "fbmk_2_fire" } ],
+    "blueprint_resources": [ "fake_fireplace", "fake_fireplace", "pot" ],
+    "components": [ [ [ "pot", 1 ], [ "rock_pot", 1 ], [ "pot_copper", 1 ], [ "clay_pot", 1 ] ], [ [ "pan", 1 ] ] ],
+    "flags": [
+      "MAP_MIRROR_HORIZONTAL_IF_N",
+      "MAP_ROTATE_90_IF_NE",
+      "MAP_MIRROR_HORIZONTAL_IF_E",
+      "MAP_ROTATE_90_IF_E",
+      "MAP_ROTATE_180_IF_SE",
+      "MAP_MIRROR_HORIZONTAL_IF_S",
+      "MAP_ROTATE_180_IF_S",
+      "MAP_ROTATE_270_IF_SW",
+      "MAP_MIRROR_HORIZONTAL_IF_W",
+      "MAP_ROTATE_270_IF_W"
+    ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "fbmk_2_well",
+    "description": "We should build a well.  This will make cooking more convenient and provides water to support a chef't garden.",
```

Comment:
What does “chef't” mean here?

Reply:
Thanks for catching it. It's the new, modern, hyper popular spelling of "chef's" ;P